resources/i18n/de/strings.xml

```diff
@@ -1,4 +1,96 @@
+
+  <!-- XTXT: Admission State - 1G_PLUS_RAT Badge Text -->
+  <string name="ADMISSION_STATE_1G_PLUS_RAT_BADGE_TEXT">"1G+"</string>
+  <!-- XTXT: Admission State - 1G_PLUS_RAT Card Subtitle -->
+  <string name="ADMISSION_STATE_1G_PLUS_RAT_SUBTITLE_TEXT">"1G+ Schnelltest"</string>
+  <!-- XTXT: Admission State - 1G_PLUS_RAT Card Body -->
+  <string name="ADMISSION_STATE_1G_PLUS_RAT_LONG_TEXT">"Ihre Zertifikate erfüllen die 1G-Plus-Regel, es sei denn, es wird ein PCR-Test benötigt. Wenn Sie Ihren aktuellen Status vorweisen müssen, schließen Sie diese Ansicht und zeigen Sie den QR-Code auf der Zertifikatsübersicht."</string>
+
+  <!-- XTXT: Admission State - 1G Badge Text -->
+  <string name="ADMISSION_STATE_1G_BADGE_TEXT">"1G"</string>
+  <!-- XTXT: Admission State - 1G Card Subtitle -->
+  <string name="ADMISSION_STATE_1G_SUBTITLE_TEXT">"1G"</string>
+  <!-- XTXT: Admission State - 1G Card Body -->
+  <string name="ADMISSION_STATE_1G_LONG_TEXT">"Ihre Zertifikate erfüllen die 1G-Regel. Wenn Sie Ihren aktuellen Status vorweisen müssen, schließen Sie diese Ansicht und zeigen Sie den QR-Code auf der Zertifikatsübersicht."</string>
+
+  <!-- XTXT: Admission State - 2G_PLUS_PCR Badge Text -->
+  <string name="ADMISSION_STATE_2G_PLUS_PCR_BADGE_TEXT">"2G+"</string>
+  <!-- XTXT: Admission State - 2G_PLUS_PCR Card Subtitle -->
+  <string name="ADMISSION_STATE_2G_PLUS_PCR_SUBTITLE_TEXT">"2G+ PCR-Test"</string>
+  <!-- XTXT: Admission State - 2G_PLUS_PCR Card Body -->
+  <string name="ADMISSION_STATE_2G_PLUS_PCR_LONG_TEXT">"Ihre Zertifikate erfüllen die 2G-Plus-Regel. Wenn Sie Ihren aktuellen Status vorweisen müssen, schließen Sie diese Ansicht und zeigen Sie den QR-Code auf der Zertifikatsübersicht."</string>
+
+  <!-- XTXT: Admission State - 2G_PLUS_RAT Badge Text -->
+  <string name="ADMISSION_STATE_2G_PLUS_RAT_BADGE_TEXT">"2G+"</string>
+  <!-- XTXT: Admission State - 2G_PLUS_RAT Card Subtitle -->
+  <string name="ADMISSION_STATE_2G_PLUS_RAT_SUBTITLE_TEXT">"2G+ Schnelltest"</string>
+  <!-- XTXT: Admission State - 2G_PLUS_RAT Card Body -->
+  <string name="ADMISSION_STATE_2G_PLUS_RAT_LONG_TEXT">"Ihre Zertifikate erfüllen die 2G-Plus-Regel, es sei denn, es wird ein PCR-Test benötigt. Wenn Sie Ihren aktuellen Status vorweisen müssen, schließen Sie diese Ansicht und zeigen Sie den QR-Code auf der Zertifikatsübersicht."</string>
+
+  <!-- XTXT: Admission State - 2G Badge Text -->
+  <string name="ADMISSION_STATE_2G_BADGE_TEXT">"2G"</string>
+  <!-- XTXT: Admission State - 2G Card Subtitle -->
+  <string name="ADMISSION_STATE_2G_SUBTITLE_TEXT">"2G"</string>
+  <!-- XTXT: Admission State - 2G Card Body -->
+  <string name="ADMISSION_STATE_2G_LONG_TEXT">"Ihre Zertifikate erfüllen die 2G-Regel. Wenn Sie Ihren aktuellen Status vorweisen müssen, schließen Sie diese Ansicht und zeigen Sie den QR-Code auf der Zertifikatsübersicht."</string>
+
+  <!-- XTXT: Admission State - 3G_WITH_PCR Badge Text -->
```

Comment:
Isn't 3G with PCR test known as 3G+?

Reply:
Thanks for catching this, fixed with https://github.com/corona-warn-app/cwa-app-ccl/pull/3/commits/2c66c0a50661c267dc146425506d409550f15180